pipelines/controller/src/test/java/org/openmrs/analytics/MockUtil.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020-2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openmrs.analytics;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+import okhttp3.mockwebserver.MockResponse;
+import org.apache.commons.io.IOUtils;
+import org.apache.http.entity.ContentType;
+
+public class MockUtil {
+
+  public static MockResponse getMockResponse(String filePath) throws IOException {
+    Objects.requireNonNull(filePath);
+    InputStream stream = ClassLoader.getSystemResourceAsStream(filePath);
```

Comment:
Consider using Guava's utility methods, e.g., Resources.getResource like [here](https://github.com/google/fhir-data-pipes/blob/45390c1ad484e9503d146d895d0102f49944b67c/pipelines/batch/src/test/java/org/openmrs/analytics/ReadJsonFilesTest.java#L68). And I think you can get rid of the rest of this method too.

Reply:
Done